src/OpenLoco/src/Windows/ObjectSelectionWindow.cpp

```diff
@@ -52,10 +54,65 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
     static constexpr int kRowHeight = 12;
     static constexpr Ui::Size kWindowSize = { 600, 398 };
 
-    static loco_global<uint8_t[999], 0x004FE384> _4FE384;
+    enum class ObjectTabFlags : uint8_t
+    {
+        none = 0U,
+        alwaysHidden = 1U << 0,
+        advanced = 1U << 1,
+        hideInGame = 1U << 2,
+        hideInEditor = 1U << 3,
+        showEvenIfSingular = 1U << 4,
+    };
+    OPENLOCO_ENABLE_ENUM_OPERATORS(ObjectTabFlags);
+
+    struct TabDisplayInfo
+    {
+        string_id name;
+        uint32_t image;
+        ObjectTabFlags flags;
+    };
+
+    // clang-format off
+    static constexpr std::array<TabDisplayInfo, ObjectManager::maxObjectTypes> _tabDisplayInfo = {
```

Comment:
Not sure why the brace initializer is failing here. I might have to invoke the type for each of the initialiser braces/tuples? I’d rather not; that’d be ugly and should be redundant, right?

Reply:
You seem to have on brace too many actually, it should be like
```
= {
    { StringIds::object_interface_styles, ImageIds::tab_object_settings, ObjectTabFlags::advanced },
}
```

Reply:
> Not sure why the brace initializer is failing here. I might have to invoke the type for each of the initialiser braces/tuples? I’d rather not; that’d be ugly and should be redundant, right?

You need to give it at least one with the type but at which point its just neater to do all of them with the type. I've added the type in the latest commit.